### DISCUSSION TITLE
Qfield GPS

### DISCUSSION THREAD

**Discussion by MikeC1976** [opengisch/QField#5565] [Q&A]
can someone create a qfield plugin that allows a GPS course line, the ability to use routes and waypoint alarms

**Comment by MMR100** [comment]
I don't use a plugin but I created an hyperlink in each feature form to open Google Maps. 

In case it can help, here are the steps:

Create a field in QGIS and select "attachment" in the available widgets. 

Then apply the following formula

Google Itinerary from current location: 
concat( 'https://www.google.com/maps/dir/?api=1&destination=', y(transform($geometry, layer_property(@layer, 'crs'),'EPSG:4326')), '%2C', x(transform($geometry, layer_property(@layer, 'crs'),'EPSG:4326')), '&travelmode=driving')


 
Point Google Maps:
concat( '[https://maps.google.com?q=](https://maps.google.com/?q=)  ',y(transform($geometry, layer_property(@layer, 'crs'),'EPSG:4326')), '%2C', x(transform($geometry, layer_property(@layer, 'crs'),'EPSG:4326')), '&zoom=19&t=h')
